app/controllers/katello/api/v2/repositories_controller.rb

```diff
@@ -45,7 +44,7 @@ def custom_index_relation(collection)
       param :checksum_type, String, :desc => N_("Checksum of the repository, currently 'sha1' & 'sha256' are supported")
       param :docker_upstream_name, String, :desc => N_("Name of the upstream docker repository")
       param :docker_tags_whitelist, Array, :desc => N_("Comma-separated list of tags to sync for Container Image repository")
-      param :download_policy, ["immediate", "on_demand", "background"], :desc => N_("download policy for yum repos (either 'immediate', 'on_demand', or 'background (deprecated)')")
+      param :download_policy, ["immediate", "on_demand", "background"], :desc => N_("download policy for yum repos (either 'immediate' or 'on_demand')")
```

Comment:
I think we need to remove 'background' here from the list of acceptable values (and add a migration to change the db from 'background' => 'immediate' for all RootRepository objects that are set to background